### ISSUE TITLE
Document recent changes to filmic defaults

### ISSUE THREAD

**Issue by elstoc** [darktable-org/dtdocs#527]
See https://github.com/darktable-org/darktable/commit/9a82b2f1584626684142e3b441a631ded2a9f467 for details of the changes made

**Comment by gi-man** [comment]
Yes the defaults were changed, but I dont think this belongs in the manual. I searched in the current manual and I could not find a place that describes the default values. If anything, this changes should just be in the release notes only.

**Comment by elstoc** [comment]
The user manual does mention defaults and states things that contradict with the new defaults. For example, it says that the "safe mode" is default and it suggests that maximising the latitude is a good thing, whereas the new default latitude is as low as it can get.
